charts/openinwoner/values.yaml

```diff
@@ -166,7 +166,6 @@ extraVolumeMounts: []
 settings:
   allowedHosts: ""
   djangoSettingsModule: open_inwoner.conf.docker
-  useXForwardedHost: true
```

Comment:
is dit niet standaard nodig wanneer je achter een reverse proxy zit? Ik zie het bij de andere componenten (behalve openforms) op true staan.

Reply:
Nodig is het blijkbaar niet, voor de werking. Zie [openzaak_envvars](https://open-zaak.readthedocs.io/en/latest/installation/config/env_config.html?highlight=USE_X_FORWARDED_HOST#optional)

Security ding.

Dus dat zijn envvars in openforms / openzaak (andere niet gechecked). Openinwoner heeft hem  niet, dus denk niet dat het zin heeft om deze te zetten. De value werd dus alleen gebruikt in nginx config, maar zonder de envvar doet het weinig.